pkg/web/auth.go

```diff
@@ -341,3 +342,76 @@ func (s *Server) reauthenticateAPIKey(c *gin.Context, keyID ulid.ULID) (_ *auth.
 
 	return auth.NewClaims(ctx, apikey)
 }
+
+func (s *Server) ChangePassword(c *gin.Context) {
```

Comment:
This handler allows the logged in user to change their own password, regardless of their role.